yarn.lock

```diff
@@ -12796,6 +12990,37 @@ rc-table@~7.11.0:
     rc-util "^5.4.0"
     shallowequal "^1.1.0"
 
+<<<<<<< HEAD
```

Comment:
Seems like we  need to run `yarn install` locally so that these conflicts can be resolved